doc/neps/nep-0046-sponsorship-guidelines.rst

```diff
@@ -0,0 +1,254 @@
+:Created: 2020-12-27
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+
+
+Abstract
+--------
+
+This NEP provides guidelines on how the NumPy project will acknowledge
+financial and in-kind support.
+
+
+Motivation and Scope
+--------------------
+
+In the past few years the NumPy project has gotten significant financial
+support, as well as dedicated work time for maintainers to work on NumPy. There
+is a need to acknowledge that support - funders and organizations expect or require
+it, it's helpful when looking for new funding, and it's the right thing to do.
+Furthermore, having a clear policy for how NumPy acknowledges support is
+helpful when searching for new support.
+
+This NEP is aimed at both the NumPy community - who can use it when looking for
+support and acknowledging existing support - and at past, current and
+prospective sponsors, who often want or need to know what they get in return
+for their support other than a healthier NumPy.
+
+The scope of this proposal includes:
+
+- direct financial support, employers providing paid time for NumPy maintainers
+  and regular contributors, and in-kind support such as free hardware resources or
+  services.
+- where and how NumPy acknowledges support (e.g., logo placement on the website).
+- the amount and duration of support which leads to acknowledgement.
+- who in the NumPy project is responsible for sponsorship related topics, and
+  how to contact them.
+
+
+How NumPy will acknowledge support
+----------------------------------
+
+There will be two different ways to acknowledge financial and in-kind support,
+one to recognize significant active support, and another one to recognize
+support received in the past and smaller amounts of support.
+
+Entities who fall under "significant active supporter" we'll call Sponsor.
+The minimum level of support given to NumPy to be considered a Sponsor are:
+
+- $30,000/yr for unrestricted financial contributions
+- $60,000/yr for financial contributions for a particular purpose
+- $100,000/yr for in-kind contributions
+
+We define support being active as:
+
+- for a one-off donation: it was received within the previous 12 months
+- for a grant or other support with a defined start and end date: the duration
+  of a grant period.
+
+After support moves from "active" to "inactive" status, the acknowledgement
+will be left in its place for at least another 6 months. If appropriate, the
+funding team can discuss opportunities for renewal with the sponsor. After
+those 6 months, acknowledgement may be moved to the historical overview. The
+exact timing of this move is at the discretion of the funding team, because
+there may be reasons to keep it in the more prominent place for longer.
+
+The rationale for the above funding levels is that unrestricted financial
+contributions are typically the most valuable for the project, and the hardest
+to obtain.  The opposite is true for in-kind contributions. The dollar value of
```

Comment:
I would not agree with that as stated.  It could be read to imply that in-kind contributions are the least valuable.  But in-kind contributions can be extremely valuable, since the team does not have to deal with management, finances, etc.  I suspect you may have meant that in-kind contributions are easier to obtain, which is true.

Reply:
I think with in-kind contributions you and I have slightly different things in mind. I'm really thinking about "company X decides to let one of their engineers contribute to NumPy". That likely means:

- the engineer works on things most important to company X
- there's extra maintainer effort required for guidance and code review

I suspect you are mostly thinking about maintainers being paid. But that falls under Institutional Partners, which is a different thing. Time for a maintainer is inherently more valuable than time of a new/external contributor.

Reply:
Yes, so I think that really is the only part of the NEP that I think could benefit from some clarity---exactly how these relationships work: companies that contribute something they want to NumPy (costs us mentorship) vs. companies that allow maintainers to work on NumPy as part of their jobs (then we count them only if they give more than 6 months, if I recall correctly), vs groups that hire someone specifically to work on NumPy.

Reply:
I can make that explicit, I agree that would help.